doc/source/conf.py

```diff
@@ -265,7 +265,7 @@ def render_svg_logo(path):
 # documentation.
 html_theme_options = {
     "use_edit_page_button": True,
-    "announcement": None,
+    "announcement": "Ray Summit unites the community. Submit a <b><a target='_blank' href='https://raysummit.anyscale.com/flow/anyscale/raysummit2024/landing/page/eventsite??utm_source=cfpraydocs_gl=1*949o36*_ga*Mzk2ODgxMTUwLjE3MTM4OTE0NzM.*_ga_T6EXHYG44V*MTcxNDA3ODUzNC43LjAuMTcxNDA3ODUzNC42MC4wLjEyMDk3NzQ2MjE'>session proposal</a></b> to share your experience.",
```

Comment:
```suggestion
    "announcement": "Ray Summit unites the community. Submit a <b><a target='_blank' href='https://raysummit.anyscale.com/flow/anyscale/raysummit2024/landing/page/eventsite?utm_source=cfpraydocs_gl=1*949o36*_ga*Mzk2ODgxMTUwLjE3MTM4OTE0NzM.*_ga_T6EXHYG44V*MTcxNDA3ODUzNC43LjAuMTcxNDA3ODUzNC42MC4wLjEyMDk3NzQ2MjE'>session proposal</a></b> to share your experience.",
```

maybe? @angelinalg ? 

@peytondmurray where do you get this URL from?

Reply:
@angelinalg passed this to me.